distribution/src/main/assembly/bin.xml

```diff
@@ -82,6 +82,13 @@
                 <exclude>*/**</exclude>
             </excludes>
         </fileSet>
+
+        <!-- Copying the in-place update tools-->
+        <fileSet>
```

Comment:
We don't update toolkit via wum. For toolkit updates we do a release and host the dist in website. So I don't think we need to add update tool here

Reply:
Will send another PR with this.